main_app/templates/order_detail.html

```diff
@@ -47,7 +47,7 @@ <h4>Shipped to:</h4>
       </tr>
     </tfoot>
   </table>
-  {% if order.status == 0 or order.status == 1 %}
+  {% if order.status == order.Status.CURRENT or order.status == order.Status.IN_PROGRESS %}
```

Comment:
I think we should refer to constants using the class instead of a specific instance since it removes the need for the existence of the instance – possibly less error prone
ie:
Use `Cart.Status` instead of `order.Status`